diff --git a/node_modules/@dilsonspickles/components/dist/Flyout.js b/node_modules/@dilsonspickles/components/dist/Flyout.js
new file mode 100644
--- /dev/null
+++ b/node_modules/@dilsonspickles/components/dist/Flyout.js
@@ -0,0 +1,6 @@
+/* attach-css-imports: managed */
+require('./Flyout.css');
+"use strict";Object.defineProperty(exports, "__esModule", {value: true});
+var _chunkFLYOUTjs = require('./chunk-FLYOUT.js');
+exports.Flyout = _chunkFLYOUTjs.Flyout;
+
